server/sonar-web/src/main/js/components/SourceViewer/SourceViewerBase.tsx

```diff
@@ -331,26 +308,30 @@ export default class SourceViewerBase extends React.PureComponent {
     }
     const firstSourceLine = this.state.sources[0];
     const lastSourceLine = this.state.sources[this.state.sources.length - 1];
-    this.props
-      .loadIssues(
-        this.props.component,
-        firstSourceLine && firstSourceLine.line,
-        lastSourceLine && lastSourceLine.line
-      )
-      .then(issues => {
+    this.props.loadIssues!(
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Forbidden non null assertion ([typescript:S2966](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS2966))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fpopups&issues=AWHb6qYww6XGHKvPuQTF&open=AWHb6qYww6XGHKvPuQTF)

[ISSUE_KEY]: AWHb6qYww6XGHKvPuQTF